src/autosave.js

```diff
@@ -62,6 +64,12 @@ export default class Autosave extends Plugin {
 	constructor( editor ) {
 		super( editor );
 
+		const config = editor.config.get( 'autosave' ) || {};
+
+		// A minimum amount of time that needs to pass after the last action.
+		// After that time the provided save callbacks are being called.
+		const waitingTime = config.waitingTime || 2000;
```

Comment:
https://github.com/ckeditor/ckeditor5-autosave/issues/10#issuecomment-408118358

Reply:
> After #12 (comment) default timeout should work in most cases and is should not need to be configurable.

Reply:
Also, I am not sure if 2000 is not too much. Maybe 1000?

Reply:
> After #12 (comment) default timeout should work in most cases and it should not need to be configurable.

Right, I overlooked it.

> Also, I am not sure if 2000 is not too much. Maybe 1000?

The last call will be on the `Editor#destroy` or `Window#beforeunload`, so IMO we can use a higher value here.